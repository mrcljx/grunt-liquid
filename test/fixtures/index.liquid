{% extends layout %}

{% block body %}
  HELLO!

  {% include products %}
{% endblock %}