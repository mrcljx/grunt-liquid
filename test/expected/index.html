<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8">
<title>Document</title>
  </head>
  <body>
    
  HELLO!

  <ul id="products">
  
    <li>
      <h2>Wonderflonium</h2>
      Only $9.99

      Great for building freeze rays!
    </li>
  
</ul>

    <script src="script.js"></script>
  </body>
</html>